BibCnrs/BibCnrs
# Accessiblity page

## Problem
We lack an accessibilitypage

## Solution

Add an accessibility page ediatble from the admin

## How to Test

- Go to the admin "gesion de contenu" section
- create a new content with accessibility page chip
- go to home
- click on the accessibility link in the footer
- see the contenyou created on the admin

## Preview

![image](https://github.com/user-attachments/assets/510c6e00-4c83-4949-a0e7-eb2eb11832da)
![image](https://github.com/user-attachments/assets/ed9f5ada-7f42-4a3e-abd0-788c504a4df8)
